cypress/support/step_definitions/dictionary/addBulkConceptsToDictionary/addBulkConceptsToDictionary.ts

```diff
@@ -0,0 +1,141 @@
+/// <reference types="cypress" />
+/// <reference types="../../../" />
+import { 
+  After, 
+  Before, 
+  Given, 
+  Then, 
+  When } from "cypress-cucumber-preprocessor/steps";
+import { getDictionaryId, getUser } from '../../../utils';
+
+Given("the user is on the view dictionary concepts page", () => {
+  cy.visit(`/users/${getUser()}/collections/${getDictionaryId()}/concepts/`);
+  cy.url().should(
+    "contain",
+    `/users/${getUser()}/collections/${getDictionaryId()}/concepts/`
+  );
+});
+
+Given('the user is on the "Add concepts in bulk from CIEL" page', () => {
+  cy.visit(
+    `/users/${getUser()}/collections/${getDictionaryId()}/add/?fromSource=CIEL/`
+  );
+  
+  cy.url().should(
+    "contain",
+    `/users/${getUser()}/collections/${getDictionaryId()}/add/?fromSource=CIEL/`
+  );
+});
+
+Given('CIEL concept "1000" is already in the dictionary', () => {
+  cy.getConcept(`/users/${getUser()}/collections/${getDictionaryId()}/concepts/1000`).should("exist");
+});
+
+When('the user clicks the "Add concepts" floating action button', () => {
+  cy.findByTitle("Add concepts").click();
+});
+
+When('the user clicks the "Add concepts" MuiButton-label button', () => {
+  cy.findByText("Add concepts").click();
+});
```

Comment:
We definitely shouldn't have the text "MuiButton-label button" here. It should probably be `'the user clicks the "Add concepts" button'`.